operations/boot_orchestration/Clean_Up_After_a_BOS-BOA_Job_is_Completed_or_Cancelled.md

```diff
@@ -40,140 +43,151 @@ When a session is launched, the items below are created:
+    - Configuration has completed and the desired state needs to be cleaned up so that configuration does not happen on restart:
 
-            To find the impacted component names (xnames) for the components with the desired configuration matching what was applied:
+        1. (`ncn-mw#`) Unset the desired state for all affected components.
 
-            ```bash
-            cray cfs components list
-            ```
+            1. Find the component names (xnames) for the components with the desired configuration matching what was applied.
 
-            Prevent the configuration from running:
+                ```bash
+                cray cfs components list
+                ```
 
-            ```bash
-            cray cfs components update XNAME --desired-state-commit
-            ```
+            1. Prevent the configuration from running.
 
-            This needs to be done for each component. It is enough to prevent configuration from running, and it does not revert to the previous desired state. The previous desired state has already been overwritten at this point, so if the user is trying to completely revert, they will either need to know and apply the previous desired state manually, or return BOS with the previous template using the configure operation \(which may also trigger a configure operation\).
+                ```bash
+                cray cfs components update XNAME --desired-state-commit
+                ```
 
-    -   Configuration was set/started and needs to be cancelled:
-        1.  Unset the desired state for all components affected.
+            This needs to be done for each component. It is enough to prevent configuration from running, and it does not revert to the previous desired state. The previous desired state has
+            already been overwritten at this point, so if the user is trying to completely revert, they will either need to know and apply the previous desired state manually, or create a BOS
+            session with the previous template using the `configure` operation.
 
-            To find the impacted component names (xnames) for the components with the desired configuration matching what was applied:
+    - Configuration was set/started and needs to be cancelled:
 
-            ```bash
-            cray cfs components list
-            ```
+        1. (`ncn-mw#`) Unset the desired state for all components affected.
 
-            Prevent the configuration from running:
+            1. Find the impacted component names (xnames) for the components with the desired configuration matching what was applied.
 
-            ```bash
-            cray cfs components update XNAME --desired-state-commit
-            ```
+                ```bash
+                cray cfs components list
+                ```
+
+            1. Prevent the configuration from running.
+
+                ```bash
+                cray cfs components update XNAME --desired-state-commit
```

Comment:
Same command here again -- I think this isn't correct. Or anyway, it isn't listed in the CLI help. 

I know this isn't the main purpose of your PR, but we should fix this while we're here.

Reply:
```suggestion
                cray cfs components update XNAME --desired-config ""
```